### PR TITLE
#74 Prevent changes to IA dates after assessments were created

### PR DESCRIPTION
**Pull request by mbrookeswebdev**

### CODE REVIEW

File: src/components/Form/DateInput.vue [hunk 1]
```diff
@@ -143,6 +146,10 @@ export default {
   },
   extends: FormField,
   props: {
+    assessmentsInitialised: {
```
Comment (warrensearle) [inline src/components/Form/DateInput.vue:149]: I think you could call this prop `disabled` which will then mean that the `DateInput` component implements the html `disabled` attribute. Which would be nice and something we might roll out to other form components 👍

**Review by warrensearle — CHANGES_REQUESTED** [review]
This is great. Just see the suggested prop name change 👍

File: src/components/Form/DateInput.vue (hunk 1 above)
Comment (mbrookeswebdev) [reply src/components/Form/DateInput.vue]: Done! 👍

**Review by warrensearle — APPROVED** [review]
Lovely 👍

**Pull request merged**